paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -95,9 +96,8 @@ internal abstract class BaseSheetViewModel(
 
     protected var mostRecentError: Throwable? = null
 
-    @VisibleForTesting
-    internal val _isGooglePayReady = savedStateHandle.getLiveData<Boolean>(SAVE_GOOGLE_PAY_READY)
-    internal val isGooglePayReady: LiveData<Boolean> = _isGooglePayReady.distinctUntilChanged()
```

Comment:
Was this `distinctUntilChanged` important?

Reply:
StateFlow does this under the hood.